app/test_cmd.go

```diff
@@ -9,7 +9,8 @@ import (
 )
 
 type testCmd struct {
-	Pkg []string `arg:"" required:"" help:"Run sanity tests for these packages."`
+	Pkg          []string `arg:"" required:"" help:"Run sanity tests for these packages."`
+	CheckSources bool     `help:"Check that package sources are reachable" default:"true"`
```

Comment:
Maybe add [negatable:""](https://github.com/alecthomas/kong#supported-tags) so we get `--[no-]check-sources`?

Reply:
Nice! Will switch to that